pyproject.toml

```diff
@@ -268,7 +268,9 @@ exclude = [
   "tools",
   "docs",
   "tests/examples",
-  "tests/docker_service_fixtures.py"
+  "tests/docker_service_fixtures.py",
+  "litestar/contrib/pydantic/pydantic_dto_factory.py",
```

Comment:
pyright complains about every instance where a pydantic import is used here because for some reason it think that it's a "variable"

Reply:
Funny story. I was removing `# pyright: ignore`'s in that module to see what pyright was complaining about as I was going through the review, and it would never report an issue. So I convinced myself that something was wrong with my pyright installation :D Perhaps we can remove all those ignore directives then?